.circleci/config.yml

```
@@ -90,8 +90,9 @@ commands:
             # Install
             . /opt/conda/etc/profile.d/conda.sh
             mkdir /logs
-            mamba env create >> /logs/conda.txt 2>&1
+            mamba env create
             conda activate esmvaltool
+            conda list >> /logs/conda.txt
```

```suggestion
            mamba list >> /logs/conda.txt
```

Use mamba whenever possible?

---

that'll do, Piglet, that'll do :grin: